Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -591,24 +559,122 @@ public IEnumerator TestDisabledOnStart()
 
             Assert.True(wasClicked, "Interactable was not clicked.");
 
+            // Cleanup
             GameObject.Destroy(interactable.gameObject);
         }
 
+        /// <summary>
+        /// Test the TriggerOnClick API for Interactable both when IsEnabled and not. Button should fire OnClick and move UI
+        /// </summary>
         [UnityTest]
+        public IEnumerator TestTriggerOnClick()
+        {
+            Interactable interactable;
+            Transform innerCylinderTransform;
+
+            InstantiatePressButtonPrefab(
+                new Vector3(0.0f, 0.0f, 0.5f),
+                DefaultRotation,
+                out interactable,
+                out innerCylinderTransform);
+
+            // Subscribe to interactable's on click so we know the click went through
+            bool wasClicked = false;
+            interactable.OnClick.AddListener(() => { wasClicked = true; });
+
+            Vector3 targetStartPosition = innerCylinderTransform.localPosition;
+
+            //
+            // Test TriggerOnClick when disabled
+            //
+            interactable.IsEnabled = false;
+
+            interactable.TriggerOnClick();
+
+            Assert.False(wasClicked, "Interactable was clicked.");
+            Assert.False(interactable.IsVisited, "Interactable was visited.");
+
+            //
+            // Test TriggerOnClick when enabled
+            //
+            interactable.IsEnabled = true;
+
+            interactable.TriggerOnClick();
+            yield return new WaitForSeconds(ButtonReleaseAnimationDelay);
+
+            Assert.True(wasClicked, "Interactable was not clicked.");
+            Assert.True(interactable.IsVisited, "Interactable was not visited.");
+
+            // Cleanup
+            GameObject.Destroy(interactable.gameObject);
+        }
+
+        /// <summary>
+        /// Tests that radial buttons can be selected and deselected, and that a radial button
+        /// set allows just one button to be selected at a time
+        /// </summary>
+        [UnityTest]
+        public IEnumerator TestRadialSetPrefab()
+        {
+            var radialSet = InstantiateInteractableFromPath(Vector3.forward, Quaternion.identity, RadialSetPrefabAssetPath);
+            var firstRadialButton = radialSet.transform.Find("Radial (1)").GetComponent<Interactable>();
+            var secondRadialButton = radialSet.transform.Find("Radial (2)").GetComponent<Interactable>();
+            var thirdRadialButton = radialSet.transform.Find("Radial (3)").GetComponent<Interactable>();
+            var testHand = new TestHand(Handedness.Right);
+            yield return testHand.Show(Vector3.zero);
+
+            Assert.IsTrue(firstRadialButton.IsToggled);
+            Assert.IsFalse(secondRadialButton.IsToggled);
+            Assert.IsFalse(thirdRadialButton.IsToggled);
+
+            yield return testHand.Show(Vector3.zero);
```

Comment:
Nit: The hand is already shown right now - I'm guessing this line is included for readability right?

Reply:
The test was originally written with that line and I just missed it myself, so just an oversight. Fixing now. Good catch!